leapp/snactor/commands/repo/__init__.py

```diff
@@ -1,3 +1,4 @@
+from __future__ import print_function
```

Comment:
Shouldn't there be a blank line between imports from `__future__` and other imports?